terraform/10-video-hearing-wowza/modules/wowza/network.tf

```diff
@@ -47,4 +55,30 @@ resource "azurerm_network_security_group" "wowza" {
     destination_address_prefix = "*"
   }
   tags = var.tags
+}
+
+resource "azurerm_virtual_network_peering" "vh-to-hub" {
+  provider = azurerm.peering_client
+  for_each                     = toset(local.peering_vnets)
+
+  name                         = var.service_name
```

Comment:
Would you not want this called the same as the VNet it is connecting two, so you know what VNet it is?